sensor/common/registry/registry_store_test.go

```diff
@@ -356,10 +410,10 @@ func TestRegistryStore_CentralIntegrations(t *testing.T) {
 // TestRegistryStore_CreateImageIntegrationType verifies the type of an image integration
 // is properly set based on the provided registry.
 func TestRegistryStore_CreateImageIntegrationType(t *testing.T) {
-	ii := createImageIntegration("http://example.com", config.DockerConfigEntry{}, "")
+	ii := createImageIntegration("http://example.com", config.DockerConfigEntry{}, "http://example.com")
 	assert.Equal(t, ii.Type, types.DockerType)
 
-	ii = createImageIntegration("https://registry.redhat.io", config.DockerConfigEntry{}, "")
+	ii = createImageIntegration("https://registry.redhat.io", config.DockerConfigEntry{}, "https://registry.redhat.io")
 	assert.Equal(t, ii.Type, types.RedHatType)
 }
```

Comment:
Curious, why add the name(s) here?

Reply:
I added it to the other `createImageIntegration` calls where it is needed to avoid collisions. So I thought it looked more consistent and perhaps a bit more realistic test case.